extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -92,7 +92,7 @@ export class RemoteQueriesManager {
 
     return {
       executionEndTime,
-      analysisResults,
+      analysisSummaries: analysisResults,
```

Comment:
As above, maybe `analysisResults` -> `analysisSummaries` would be more consistent?